cloud/deployment/deployment.go

```diff
@@ -545,7 +517,60 @@ func selectDeployment(deployments []astro.Deployment, message string) (string, e
 	choice := input.Text("\n> ")
 	selected, ok := deployMap[choice]
 	if !ok {
-		return "", errInvalidDeploymentKey
+		return astro.Deployment{}, errInvalidDeploymentKey
+	}
+	return selected, nil
+}
+
+func GetDeployment(ws, deploymentID string, client astro.Client) (astro.Deployment, error) {
```

Comment:
we would need unit test for this function

Reply:
I added unit tests but. It is still reporting loss in coverage. I think it is because I deleted a lot of code